Backpack/Button/Classes/BPKButton.m

```diff
@@ -301,21 +307,30 @@ - (void)updateBackgroundAndStyle {
                 break;
             }
             case BPKButtonStyleSecondary: {
-                [self setBorderedStyleWithColor:BPKColor.blue500];
+                [self setBorderedStyleWithColor:BPKColor.gray100];
                 if (self.isHighlighted) {
                     self.gradientLayer.gradient = [self gradientWithSingleColor:highlightedWhite];
                     [self.gradientLayer setNeedsDisplay];
                 }
                 break;
             }
             case BPKButtonStyleDestructive: {
-                [self setBorderedStyleWithColor:BPKColor.red500];
+                [self setBorderedStyleWithColor:BPKColor.gray100];
                 if (self.isHighlighted) {
                     self.gradientLayer.gradient = [self gradientWithSingleColor:highlightedWhite];
                     [self.gradientLayer setNeedsDisplay];
                 }
                 break;
             }
+            case BPKButtonStyleOutline: {
+                [self setBorderedStyleWithColor:BPKColor.white withGradientColor:BPKColor.clear];
+                if (self.isHighlighted) {
```

Comment:
`setBorderedStyleWithColor` is getting called twice here if `isHighlighted == true`, might be more efficient to do the first one in an `else` block

Reply:
Nah :)

Reply:
![anigif_sub-buzz-19873-1543319955-12](https://user-images.githubusercontent.com/73652/49233814-bccb3480-f3ee-11e8-97f0-b0f767c2ee55.gif)